src/js/containers/Menu.js

```diff
@@ -8,17 +9,12 @@ const mapStateToProps = (state) => {
     var menu = state.ui[activeApp].menu
     var theme = state.theme
     var data = menu.map((command) => {
-        var dataClass = null
-        if (command.cmdIcon) {
-            dataClass = 'with-icon'
```

Comment:
Was the with-icon css class not being used?

Reply:
it was being used to decide between <svg> and <img>, now this logic is done by the imageType

Reply:
github cut out my html from the comment. this class was used to differentiate between `<svg>` and `<img>` but now imageType is used